### PR TITLE
add console.error to help troubleshoot and toast when detecting multiple wallet extensions

### PR DESCRIPTION
**Pull request by marc-aurele-besner**
## add console.error to help troubleshoot and toast when detecting multiple wallet extensions

image

### CODE REVIEW

File: src/hooks/useConnect.ts [hunk 1]
```diff
@@ -117,6 +119,16 @@ export const useConnect = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  useEffect(() => {
+    if (walletsTypeSet.size > 1)
+      toast({
+        title: 'Multiple wallets extensions detected',
```
Comment (jim-counter) [inline src/hooks/useConnect.ts:125]:
```suggestion
        title: 'Multiple wallet extensions detected',
```

File: src/hooks/useConnect.ts [hunk 2]
```diff
@@ -117,6 +119,16 @@ export const useConnect = () => {
...
+        description: 'In some cases, having multiple wallets extensions enabled at the same time can cause issues.',
```
Comment (jim-counter) [inline src/hooks/useConnect.ts:126]:
```suggestion
        description: 'In some cases, having multiple wallet extensions enabled at the same time can cause issues.',
```

**Review by jim-counter — APPROVED** [review]
Works for me! Suggested a couple of small wording changes.

**Comment by EmilFattakhov** [comment]
As always, awesome solution Marc!

**Pull request merged**